src/main/java/com/salesforce/dev/framework/DriverManager.java

```diff
@@ -56,8 +56,8 @@ private void initializer(){
             DesiredCapabilities caps;
             if(browserRemote.equals("Chrome"))
                 caps = DesiredCapabilities.chrome();
-            else
-                caps = DesiredCapabilities.chrome();
+            else //firefox
+                caps = DesiredCapabilities.firefox();
```

Comment:
Consider using the setCapability instead of if else
something like this
```java
DesiredCapabilities capabilities = new DesiredCapabilities();
capabilities.setCapability(CapabilityType.PLATFORM, Environment.getInstance().getSaucePlatform());
capabilities.setCapability(CapabilityType.BROWSER_NAME, Environment.getInstance().getSauceBrowserName());
capabilities.setCapability(CapabilityType.VERSION, Environment.getInstance().getSauceBrowserVersion());
```